config/prow/cluster/monitoring/mixins/prometheus/stale_alerts.libsonnet

```diff
@@ -8,13 +8,13 @@
             alert: 'Prow images are stale',
             expr: |||
               time()-max(prow_version) > %d * 24 * 3600
-            ||| % $._config.prowImageStaleByDays,
-            'for': '5m',
+            ||| % $._config.prowImageStaleByDays.daysStale,
+            'for': '%s' % $._config.prowImageStaleByDays.hoursToTrigger,
             labels: {
               severity: 'critical',
             },
             annotations: {
-              message: '@test-infra-oncall The prow images are older than %d days.' % $._config.prowImageStaleByDays,
+              message: '@test-infra-oncall The prow images are older than %d days.' % $._config.prowImageStaleByDays.daysStale,
```

Comment:
Please remove `@test-infra-oncall` that is only appropriate for k8s-prow and only appropriate for reporting sinks that route to slack so it should be included in the receiver config https://github.com/kubernetes/test-infra/blob/a38a2c68c3701f197bbf2386746ab704d4a01e55/config/prow/cluster/monitoring/alertmanager-prow_secret.yaml#L28

Reply:
Done